eksupgrade/src/eks_get_image_type.py

```diff
@@ -16,31 +16,18 @@ def image_type(node_type: str, image_id: str, region: str) -> Optional[str]:
     """Return the image location."""
     ec2_client = boto3.client("ec2", region_name=region)
     node_type = node_type.lower()
+    filters = [
+        {"Name": "is-public", "Values": ["true"]},
+    ]
 
     if node_type == "amazon linux 2":
-        filters = [
-            {"Name": "owner-id", "Values": ["602401143452"]},
-            {"Name": "name", "Values": ["amazon-eks-node-*"]},
-            {"Name": "is-public", "Values": ["true"]},
-        ]
+        filters.append({"Name": "name", "Values": ["amazon-eks-node-*"]})
     elif node_type == "ubuntu":
-        filters = [
-            {"Name": "owner-id", "Values": ["099720109477"]},
-            {"Name": "name", "Values": ["ubuntu-eks/k8s_*"]},
-            {"Name": "is-public", "Values": ["true"]},
-        ]
+        filters = {"Name": "name", "Values": ["ubuntu-eks/k8s_*"]}
```

Comment:
Should this be an `append()` or re-declaration of `filters` as a list?

```suggestion
        filters.append({"Name": "name", "Values": ["ubuntu-eks/k8s_*"]})
```

Reply:
You are right. Thank you, for catching that. Updated.